corgi/core/migrations/0103_get_latest_repo_names_from_pyxis.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.20 on 2023-11-05 23:59
+
+from django.db import migrations
+
+from corgi.core.models import Component, SoftwareBuild
+from corgi.tasks.pyxis import slow_update_name_for_container_from_pyxis
+
+
+def get_latest_repo_names_from_pyxis(apps, schema_editor) -> None:
+    brew_container_nvrs = (
+        Component.objects.filter(
+            type=Component.Type.CONTAINER_IMAGE, software_build__build_type=SoftwareBuild.Type.BREW
+        )
+        .values_list("nvr", flat=True)
+        .iterator()
+    )
+    for nvr in brew_container_nvrs:
+        slow_update_name_for_container_from_pyxis.delay(nvr)
```

Comment:
The slow_update_name_for_container_from_pyxis handles setting the name on child containers because it does a look up by NVR, which is the same for the image index and it's arch specific containers. I did add an exclude filter for the new Container Component meta_attr field `name_from_label_raw` to avoid this migration taking any action on any container builds ingested after the associated change in the Brew collector.